build.gradle

```
@@ -70,6 +70,8 @@ allprojects {
         testRuntimeOnly("org.junit.platform:junit-platform-launcher:${versions.junit5PlatformLauncher}") {
             because 'allows tests to run from IDEs that bundle older version of launcher'
         }
+
+        api "org.scala-lang.modules:scala-collection-compat${scalaSuffix}:${versions.scalaCollectionCompat}"
```

```suggestion
        api "org.scala-lang.modules:scala-collection-compat_%%${versions.scalaCollectionCompat}"
```

`_%%` is the placeholder used by scala multiversion plugin